libfwupdplugin/fu-intel-thunderbolt-nvm.c

```diff
@@ -558,11 +559,16 @@ fu_intel_thunderbolt_nvm_parse(FuFirmware *firmware,
 			self,
 			FU_INTEL_THUNDERBOLT_NVM_SECTION_DIGITAL,
 			FU_INTEL_THUNDERBOLT_NVM_DIGITAL_OFFSET_VERSION,
-			&version_raw,
+			&version_tmp,
 			error)) {
 			g_prefix_error(error, "failed to read version: ");
 			return FALSE;
 		}
+		/* LE: major=[15:8], minor=[7:2], reserved=[1], prod=[0] */
+		g_debug("NVM firmware type: %s", (version_tmp & 0x0001) ? "prod" : "dev");
```

Comment:
Shouldn't this be 0x0080, if the below mask is 0xff3f (and not 0xfffc)?

Reply:
> Shouldn't this be 0x0080, if the below mask is 0xff3f (and not 0xfffc)?

Thanks for the catch, will fix it.